rq/queue.py

```diff
@@ -305,13 +306,15 @@ def compact(self):
     def push_job_id(self, job_id: str, pipeline: t.Optional['Pipeline'] = None, at_front=False):
         """Pushes a job ID on the corresponding Redis queue.
         'at_front' allows you to push the job onto the front instead of the back of the queue"""
-        self.log.debug(f"Pushing job {blue(job_id)} into the queue {green(self.name)}, current job count {self.count}")
+        self.log.debug(f"Pushing job {blue(job_id)} into the queue {green(self.name)}.")
+        if self.debug_mode:
```

Comment:
Sorry, do you mind deleting this? I don't think we need this as we'll know the job count after job is pushed, sorry :(